src/components/TodoList.js

```diff
@@ -0,0 +1,45 @@
+import Compoent from "../core/Component.js";
+
+export default class TodoList extends Compoent {
+	template() {
+		// return `
+		// ${this.$props.getFilteredItem().map(({ id, text, active }) => `
+		// 	<li data-id="${id}">
+		// 	<div class="view">
+		// 	<input class="toggle" type="checkbox" checked=${active}>
+		// 	<label class="label">${text}</label>
+		// 	<button class="destroy">삭제</button>
+		// 	</div>
+		// 	<input type="text" class="edit">
+		// 	</li>
+		// 	`).join('')}
+		// `;
+
+		return `
+		${[...Object.values(this.$state.todos)].map(({id, text, active}) => `
```

Comment:
```suggestion
		${[...Object.values(this.$state.todos)].map(({id, text, active}) => `
```

${~~~} 안에 자바스크립트 코드를 넣어 바로  넘겨준 거군요! map돌린 것을 상수로 받아서 넘겨주는 방식으로 했는데 새로운 표현방식 알아갑니다~!

Object.values()메서드도 알아갑니다! 

그런데 여기서 [...Object.values] 이렇게 풀어 써주시는 이유가 뭔가요?~!  values가 배열을 반환하는 것 같아서요! 하나의 배열로 묶여있는게 아닌가 싶어서요!

Reply:
유사배열객체여서 저렇게 했었던 것으로 기억합니다.
감사합니다!

Reply:
아! 유사배열이라는 게 있군요!! 검색해 보면서 새롭게 알았어요! 답변 감사드려요~!ㅎㅎㅎ 풀어서 []안에 넣어주면 배열의 메서드들을 사용할 수 있군요! 👍